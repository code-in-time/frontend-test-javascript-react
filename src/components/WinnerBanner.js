import React, { Component } from 'react';
import PropTypes from 'prop-types';

class WinnerBanner extends Component {
  render() { 
    return ( 
      <div className="mt-4">
        Results: <strong>{this.props.result}</strong>
      </div>
     );
  }
}

WinnerBanner.defaultProps = {
  result: 'fffff',
};

WinnerBanner.propTypes = {
  result: PropTypes.string
};
 
export default WinnerBanner;