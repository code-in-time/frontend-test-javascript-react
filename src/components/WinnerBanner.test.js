import React from 'react';
import renderer from 'react-test-renderer';
import WinnerBanner from './WinnerBanner';

it('Should test the WinnerBanner.js snapshot', () => {
  const tree = renderer.create(<WinnerBanner />).toJSON()
  expect(tree).toMatchSnapshot();
})
