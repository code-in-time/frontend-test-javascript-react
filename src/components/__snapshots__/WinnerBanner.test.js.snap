// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should test the WinnerBanner.js snapshot 1`] = `
<div
  className="mt-4"
>
  Results: 
  <strong>
    fffff
  </strong>
</div>
`;
